OWML.Common/IModInputHandler.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace OWML.Common
+{
+    public interface IModInputHandler
+    {
+        RegistrationCode RegisterCombination(IModInputCombination combo);
+        RegistrationCode UnregisterCombination(IModInputCombination combo);
+        bool IsPressedExact(IModInputCombination combo);
+        bool IsNewlyPressedExact(IModInputCombination combo, bool keep = false);
+        bool WasTappedExact(IModInputCombination combo, bool keep = false);
+        bool WasNewlyReleasedExact(IModInputCombination combo, bool keep = false);
+        bool IsPressed(IModInputCombination combo);
+        bool IsNewlyPressed(IModInputCombination combo, bool keep = false);
+        bool WasTapped(IModInputCombination combo, bool keep = false);
+        bool WasNewlyReleased(IModInputCombination combo, bool keep = false);
```

Comment:
What's IsPressedExact vs IsPressed?
Maybe group these pairs?

Reply:
And what does keep do?

Reply:
exact makes it so when mod has "A" registered it won't react to "A+B" unlike ordinary version ("A+B" is not registered to any other mod in this example)

Reply:
keep being true makes it not consume "Newly" flag. I added it to somewhat mimic OW's input functionality. I wonder if I should somehow mimic Unity's input system instead

Reply:
removed keep thing, since it's probably useless and potentially confusing